rest_client.go

```diff
@@ -237,6 +237,54 @@ func (c *restClient) getLogs(id string) ([]string, error) {
 	return logs, nil
 }
 
+// DownloadMongoDB downloads the desired version of MongoDB.
+func (c *restClient) DownloadMongoDB(ctx context.Context, opts MongoDBDownloadOptions) error {
+	body, err := makeBody(opts)
+	if err != nil {
+		return err
+	}
+
+	req, err := http.NewRequest(http.MethodPost, c.getURL("/download-mongodb"), body)
+	if err != nil {
+		return errors.Wrap(err, "problem building request")
+	}
+	req = req.WithContext(ctx)
+
+	resp, err := c.client.Do(req)
```

Comment:
Does the client handle closing the response body? If not, you may have a resource leak here and in the next function.

Reply:
I added the close.